source/includes/examples/cli-example-metrics-disk.rst

```diff
@@ -0,0 +1,8 @@
+.. code-block:: 
+   :copyable: true
+
+   atlas alerts disks describe atlas-lnmtkm-shard-00-00.ajlj3.mongodb.net:27017 data \
```

Comment:
I don't recognize this command did you mean `atlas alerts describe` ? the example we have in https://www.mongodb.com/docs/atlas/cli/v1.29/command/atlas-alerts-describe/ looks like
```
atlas alerts describe 5d1113b25a115342acc2d1aa --projectId 5e2211c17a3e5a48f5497de3 --output json
```

Reply:
I see it now you wanted `atlas metrics disks describe`